### ISSUE TITLE
Custom User Roles

### ISSUE THREAD

**Issue by jcampbell05** [latera/homs#131]
We want to only allow certain users to be able to administrate orders that are assigned to them, is this possible ?

**Comment by sugarfree1** [comment]
Unfortunately, it's not possible now.
Could you give us more context, please? What is you business process and why do you need this feature?

**Comment by sugarfree1** [comment]
If I got your idea correctly then you need roles for order types. Is it so?

**Comment by jcampbell05** [comment]
@sugarfree1 roles for order location.

**Comment by sugarfree1** [comment]
Just to clarify, **location** is a custom field in your business process and it is stored in your order data. So you want to filter orders by custom fields for users. Did I get you idea correctly?

**Comment by jcampbell05** [comment]
Yes :) so User A only sees orders for User A based on a field (or even just the business logic assigning it to that user) and User B only sees orders for User B :)
